docs/reference/release-notes/highlights-7.4.0.asciidoc

```diff
@@ -0,0 +1,156 @@
+[[release-highlights-7.4.0]]
+== 7.4.0 release highlights
+++++
+<titleabbrev>7.4.0</titleabbrev>
+++++
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+[float]
+==== Results pinning
+
+You can use the new <<query-dsl-pinned-query,pinned query>>
+to define the first records
+(and the order in which they are returned)
+in a result set directly within {es}.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== New `shape` field type
+
+A new <<shape,`shape`>> field type has been added,
+which allows you to position and query shapes
+in a geometry of your choosing.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Circle ingest processor
+
+A new <<ingest-circle-processor, circle ingest processor>> has been added,
+which translates circles into regular polygons (bounded by the circles).
+This makes ingesting, indexing, searching, and aggregating circles both easy and efficient.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Aggregations on range fields
+
+The <<search-aggregations-bucket-histogram-aggregation,Histogram>>
+and <<search-aggregations-bucket-datehistogram-aggregation,Date Histogram>>
+aggregations now support the <<range,`range`>> field type.
+
+Range aggregations are useful
+when counting ranges that overlap with specific buckets
+(e.g. the number of phone calls that took place during a specific minute).
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Cumulative cardinality aggregation
+
+A new <<search-aggregations-pipeline-cumulative-cardinality-aggregation,cumulative cardinality aggregation>>
+has been added
+as part of our ongoing effort to provide advanced aggregations.
+
+You can use this new pipeline aggregation
+to calculate a net-new total of document occurrences
+within a given time range.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Snapshot lifecycle management
+
+We’re introducing <<getting-started-snapshot-lifecycle-management,snapshot lifecycle management (SLM)>>,
+which allows an administrator to define policies, 
+via API or kibana-ref/index-lifecycle-policies.html[{kib} UI],
+that manage when and how often snapshots are taken.
+You can use SLM
+to ensure that appropriate, recent backups are ready
+if disaster strikes
+or you need to restore {es} data. 
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== API key management
+
+New <<security-privileges,cluster privileges>> to manage API keys have been added,
```

Comment:
I broke this link with my earlier suggestion. This should fix it. Sorry about that.
```suggestion
New {stack-ov}/security-privileges.html[cluster privileges] to manage API keys have been added,
```